### DISCUSSION TITLE
1.2.0

### DISCUSSION THREAD

**Discussion by louislam** [louislam/dockge#187] [Announcements]
### 🆕 New Features
- #182 Add support for extra info: URLs

### Improvements
- #174 Version 3.8 will no longer be added automatically
- #60 Close terminal if there is no clients connecting
- 
### 🐞 Bug Fixes
- #186 Fix 100% cpu usage and terminal issues (Thanks @williamstein)
- #171 Fixed the status is incorrect for `dockge` stack

### 🦎 Translation Contributions
- #184 Enable an ability to translate on Weblate (Thanks @401U)
- #185 Updated Translation Guide (Thanks @Zaid-maker)
- #132 Added Polski (Thanks @Aonodensetsu)
- #154 Added Čeština (Thanks @ArtyomCZ @furkanipek)
- #159 Added Portuguese-Brazil (Thanks @altendorfme)
- #162 Added Arabic (Thanks @abosaad11)
- #164 Added Thai (Thanks @watchakorn-18k)
- #166 Added Italian(Thanks @Trainax)
- #167 Added Swedish (Thanks @annorberg98)
- #175 Added Ukrainian (Thanks @sxikari)
- #96 Added Traditional Chinese (Thanks @D3-3109)
- #184 Updated Urdu (Thanks @Zaid-maker)

### Others
- #156 Add Discussion Forms for Feature Request and Help (#156) (Thanks @Lord0fBytes)
- Other small changes, code refactoring and comment/doc updates in this repo:
@RyanBalfanz @Zaid-maker

<hr /><em>This discussion was created from the release <a href='https://github.com/louislam/dockge/releases/tag/1.2.0'>1.2.0</a>.</em>

**Comment by Zaid-maker** [comment]
Love to contribute to this project ❤️❤️

**Reply by abosaad11** [reply]
Thanks for the mention. This is my first contribution ever. I feel very proud and happy to help.

**Comment by Aonodensetsu** [comment]
feel free to mention me if new polish translations are required

**Reply by louislam** [reply]
Thank you so much! We have just setup Weblate. Now you can do this on https://weblate.kuma.pet/translate/dockge/dockge/pl/?q=state:%3Ctranslated.

**Reply by abosaad11** [reply]
Oh I didn't know about that wesbite. I just completed all the unfinished/untranslated Arabic text. If Approved Arabic should be 100%. 

BTW this website is way much better than the Github editor, just for the native support of RTL languages. So when mixing Arabic with English or some characters words wouldn't jump around forcing you to guess their position.

**Reply by louislam** [reply]
> I just completed all the unfinished/untranslated Arabic text. 

It is still accepted at this moment, because Weblate is just added recently. Feel free to create a pull request.

**Comment by barrelltitor** [comment]
What is the point of the extra info section? What do the URLs do?
